packages/engine/Source/Scene/Model/ModelDrawCommand.js

```diff
@@ -537,15 +530,10 @@ ModelDrawCommand.prototype.pushCommands = function (frameState, result) {
   }
 
   if (this._needsSkipLevelOfDetailCommands) {
-    const content = this._model.content;
-    const tileset = content.tileset;
-    const tile = content.tile;
-
-    const hasMixedContent = tileset._hasMixedContent;
-    const finalResolution = tile._finalResolution;
+    const { tileset, tile } = this._model.content;
 
-    if (hasMixedContent) {
-      if (!finalResolution) {
+    if (tileset._hasMixedContent) {
```

Comment:
I just pushed a commit to expose and document `.hasMixedContent`, since it is important for the stencil buffer in skip traversal.

However, there are at least 20 of these: undocumented private properties of `Cesium3DTileset` that are used outside the class. It would be better to document them all, but it might be out of scope for this PR?